public/app/features/alerting/unified/api/featureDiscoveryApi.ts

```diff
@@ -0,0 +1,19 @@
+import { AlertmanagerApiFeatures } from '../../../../types/unified-alerting-dto';
+
+import { alertingApi } from './alertingApi';
```

Comment:
Do we want to have this as a separate file or combine all Alertmanager-related endpoints in a single file?

Reply:
Good question. So far, the features discovery is handled a bit differently from other backend server calls, that's why I put it into a separate API. Although, I don't have any strong feelings about where it should belong